### ISSUE TITLE
"no matching manifest for linux/amd64 in the manifest list entries" when pulling latest container builds

### ISSUE THREAD

**Issue by rlgomes** [docker-library/httpd#94]
```
looker@Rodney-Gomes-Macbook-Pro.local> docker pull httpd:2.4
2.4: Pulling from library/httpd
no matching manifest for linux/amd64 in the manifest list entries
```

as stated by @kheifer one easy workaround is to pull an older version
```
looker@Rodney-Gomes-Macbook-Pro.local> docker pull httpd:2.4.29-alpine
2.4.29-alpine: Pulling from library/httpd
...
```

**Comment by ygornascimento** [comment]
same error here: Ygors-MBP$ docker pull httpd
Using default tag: latest
latest: Pulling from library/httpd
no matching manifest for linux/amd64 in the manifest list entries

**Comment by altheaschutte** [comment]
Same for me. 
```
Unable to find image 'httpd:latest' locally
latest: Pulling from library/httpd
docker: no matching manifest for linux/amd64 in the manifest list entries.
```
I noticed the latest tag had been updated an hour ago, I used the tag 2.4.29 which was updated 5 days ago and it worked. `docker pull httpd:2.4.29` Hope this gets fixed soon.

**Comment by magic-chenyang** [comment]
Same for me
```
root@VM-0-15-ubuntu:~# docker pull httpd:latest
latest: Pulling from library/httpd
no supported platform found in manifest list
```

**Comment by TheUglyHack** [comment]
This appears to be caused by a bad/hung build.

https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/httpd/

Seems to me you wouldn't remove the previous build until the new build is successful.

**Comment by sanglt** [comment]
We have same problem too.

**Comment by dconroy** [comment]
+1

**Comment by 0xlen** [comment]
+1

```bash
$ docker pull httpd
Using default tag: latest
latest: Pulling from library/httpd
no matching manifest for linux/amd64 in the manifest list entrie
```

**Comment by ThomasHartz** [comment]
https://github.com/docker-library/official-images/issues/3835
I guess the issues could be related.
TLDR from @tianon Not all architectures are created equal, but perhaps even more importantly, not all build servers we have access to are equal in performance, power, or ability to process builds reliably.

**Comment by garfieldjty** [comment]
+1

**Comment by maltyxx** [comment]
+1

**Comment by kheifer** [comment]
A temporary fix for me was to roll back to an earlier version.

**Comment by yosifkit** [comment]
Duplicate: https://github.com/docker-library/httpd/issues/93
